plugin/laravel.vim

```diff
@@ -101,23 +101,25 @@ augroup laravel_completion
   autocmd!
   " Set up nvim-completion-manager sources
   " :help NCM-source-examples
-  autocmd User CmSetup call cm#register_source({'name' : 'Laravel Route',
-        \ 'abbreviation': 'Route',
+  au User Ncm2Plugin call ncm2#register_source({
```

Comment:
Here and elsewhere, please always spell out the full command name.

```suggestion
  autocmd User Ncm2Plugin call ncm2#register_source({
```

Reply:
Understood! I haven't ever made a completion source and I was taking the shortest path to victory here, which meant copy/paste from the example in the docs and changing the necessary keys and values.hhhhh